src/docs/reference/modules/Admin/README.md

```diff
@@ -17,7 +17,7 @@ If you want to specify another prefix in the urls to access the admin section, y
 
 By default, OrchardCore logo and site name are displayed in the top navbar.
 
-You can change it by overriding 'AdminBranding' shape, either from a [custom admin theme](../../../guides/create-admin-theme/README.md) or using Admin Templates feature.
+You can change it by overriding 'AdminBranding' shape, either from a [custom admin theme](../../../guides/create-admin-theme/README.md) or using Admin Templates feature.  
```

Comment:
Remove extra whitespace at the end

Reply:
That is on purpose: Double spaces force to the next sentence to be on a new line.

It currently lokks like this:
![image](https://github.com/OrchardCMS/OrchardCore/assets/703248/fc61218c-1b65-4ae5-9ab9-280ca53a25e8)

Reply:
Strange!!